screens/shoppingList

```diff
@@ -0,0 +1,286 @@
+import React from 'react';
+import { StyleSheet, Text, View, Dimensions, Image } from 'react-native';
+import FontLoad from './components/FontLoad';
+
+export default class ShoppingList extends React.Component {
+
+
+  render() {
+    return (
+      this.state.fontLoaded ? (
+        <View style={styles.container}>
```

Comment:
This page will scroll since there's going to be multiple results.

Change this to `<ScrollView>`